doc/release-notes.md

```diff
@@ -66,6 +66,16 @@ New RPCs
 Build System
 ------------
 
+Files
+-----
+
+* On startup, the list of banned hosts and networks (via `setban` RPC) in
+  `banlist.dat` is ignored and only `banlist.json` is considered. Bitcoin Core
+  version 22.x is the only version that can read `banlist.dat` and also write
+  it in `banlist.json`. If `banlist.json` already exsits, version 22.x will not
```

Comment:
exsits => exists

and maybe "write it to" instead of "write it in" ?

Reply:
Done, thx